jiant/scripts/download_data/dl_datasets/files_tasks.py

```diff
@@ -1012,3 +1036,70 @@ def download_ropes_data_and_write_config(
         },
         path=task_config_path,
     )
+
+
+def download_acceptability_judgments_data_and_write_config(
+    task_name: str, task_data_path: str, task_config_path: str
+):
+    name_map = {
+        "acceptability_definiteness": "definiteness",
+        "acceptability_coord": "coordinating-conjunctions",
+        "acceptability_whwords": "whwords",
+        "acceptability_eos": "eos",
+    }
```

Comment:
all caps and and possibly move to constant.py for visibility

Reply:
I'm not sure if it's worth making a constant for this since this is a single-use map. I've push a slight tweak to this though.